_posts/0000-01-02-marimiracca.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+
```

Comment:
"If you want to find the secrets of the universe, think in terms of energy, frequency, and vibration! - N. Tesla"